dev-libs/tree-sitter-tsq/tree-sitter-tsq-0.19.0.ebuild

```diff
@@ -0,0 +1,15 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+TS_PV=${PV}
```

Comment:
Huh, the other ebuilds don't have this, but this one does?

If it is truly needed, maybe add a comment explaining why.

Reply:
this is because eclass assumes `TS_PV=v${PV}` (note on `v`), while this grammar somewhy don't use `v` in tags naming :shrug: 

And does comment really needed if the purpose of `TS_PV` in eclass is exactly for such cases?

Reply:
Well, although, I'll add comment anyway. And a warning to take an eye on tag names, in case if upstream will start adding "v" in the future.

Reply:
done